pkg/oc/bootstrap/docker/host/host.go

```diff
@@ -66,7 +66,7 @@ func (h *HostHelper) CanUseNsenterMounter() (bool, error) {
 		Bind("/:/rootfs:ro").
 		Entrypoint("/bin/bash").
 		Command("-c", cmdTestNsenterMount).Run()
-	return err == nil && rc == 0, nil
+	return rc == 0, err
```

Comment:
You can have an empty error and a non-zero return code.

Reply:
in that case won't set the nsenter?

Reply:
I don't know which way is more likely to work.